perun-db/addSPRegServiceAccountPolicies

```diff
@@ -0,0 +1,182 @@
+	"urn:perun:facility:attribute-def:def:rpInformationURL",
+	"urn:perun:facility:attribute-def:def:rpPrivacyPolicy",
+	"urn:perun:facility:attribute-def:def:rpAdminContact",
+	"urn:perun:facility:attribute-def:def:rpSupportContact",
+	"urn:perun:facility:attribute-def:def:rpServiceAccessTerms",
+	"urn:perun:facility:attribute-def:def:rpServiceIsNonEinfraUserEnabled",
+	"urn:perun:facility:attribute-def:def:rpServiceGrantTermsUrl",
+	"urn:perun:facility:attribute-def:def:rpOrganizationServiceGarant",
+);
+my ($user, $pwd, $overwrite);
+
+GetOptions ("help|h" => sub { print help(); exit 0;},
+	"user|u=s" => \$user,
+	"password|w=s" => \$pwd,
+	"overwrite|o" => \$overwrite) || die help();
+
+if (!defined $user) { print "[ERROR] Username for Perun DB is required! Use --help | -h to print help.\n"; exit 1; }
+if (!defined $pwd) { print "[ERROR] Password for Perun DB is required! Use --help | -h to print help.\n"; exit 1; }
+
+my $dbh = DBI->connect('dbi:Pg:dbname=perun',$user,$pwd,{RaiseError=>1,AutoCommit=>0,pg_enable_utf8=>1}) or die EPERM," Connect";
+
+my $sthGetAttributeId = $dbh->prepare(q{select a.id from attr_names a where a.attr_name=?;});
+
+my $sthNewCollection = $dbh->prepare(q{insert into attribute_policy_collections (id, attr_id, action) values (?,?,?);});
+
+my $sthNewPolicy = $dbh->prepare(q{insert into attribute_policies (id, role_id, object, policy_collection_id) values (?,?,?,?);});
+
+my $sthInsertRole = $dbh->prepare(q{insert into roles (id, name) values (?,'spregapplication');});
+
+my $sthDelCollections = $dbh->prepare(q{delete from attribute_policy_collections where id=?;});
+
+my $sthExistingPolicies = $dbh -> prepare(q{select policy_collection_id from attribute_policies where role_id=?;});
+
+# *******************************
+# **     E X E C U T I O N     **
+# *******************************
+
+# Insert SPREGAPPLICATION role to DB if not created yet
+my $roleId = $dbh->selectrow_array(q{select id from roles where name='spregapplication'});
+unless ($roleId) {
+	$roleId = $dbh->selectrow_array('select nextval(\'roles_id_seq\')');
+	$sthInsertRole->execute($roleId);
+	$dbh->commit() or die $dbh->errstr;
+	print "INFO: Spregapplication role created\n";
+}
+
+# Remove all collections where any policy with SPREGAPPLICATION exists
+if ($overwrite) {
+	$sthExistingPolicies->execute($roleId);
+	while (my @row = $sthExistingPolicies->fetchrow_array()) {
+		my ($collectionId) = @row;
+		$sthDelCollections->execute($collectionId);
+	}
+	$dbh->commit() or die $dbh->errstr;
+	print "INFO: Existing policy collections successfully removed.\n";
+}
+
+# set policies for READ attributes for SPREGAPPLICATION
+foreach my $attrName (@readAttributes) {
+	$sthGetAttributeId->execute($attrName);
+
+	if ($sthGetAttributeId->rows() ne 1) {
+		# if no corresponding attribute for given name was found, skip
+		next;
+	}
+
+	my $attrId = $sthGetAttributeId->fetchrow_array();
+	my $attrType = (split ':', $attrName)[2];
+
+    if ($attrType eq "user") {
```

Comment:
Do we need this check? Shouldn't we set all of the read attributes no matter their type?